docs/index.md

```diff
@@ -79,7 +75,6 @@ be uploaded to the community cloud,
 <a href="https://tckdb.github.io/TCKDB/", target="_blank">TCKDB</a>*.
```

Comment:
lines 65-75 also need to be reformatted, right? they're shown as is in the file itself.